src/seedu/addressbook/AddressBook.java

```diff
@@ -418,6 +428,40 @@ private static String getMessageForSuccessfulAddPerson(HashMap<PersonProperty, S
         return String.format(MESSAGE_ADDED,
                 getNameFromPerson(addedPerson), getPhoneFromPerson(addedPerson), getEmailFromPerson(addedPerson));
     }
+    
+    /**
+     * Adds a person (specified by the command args) to the address book.
```

Comment:
ouch! the header comment says it is add person, while the method is edit. 
please be careful when adding new methods/functionality to existing code. 
the header comments (javadocs documentation) should reflect the new change.